code/clue.js

```diff
@@ -10,16 +10,120 @@ const mrGreen = {
   occupation: "Entrepreneur"
 }
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS THE THE WEAPONS IF YOU LIKE.
+const mrsWhite = {
+  firstName: "Mrs",
+  lastName:"White",
+  color: "white",
+  description:"She is a good actress",
+  age: 60,
+  image: "assets/white.png",
+  occupation:"Actress"
+}
+
+const profPlum = {
+  firstName: "Victor",
+  lastName:"Plum",
+  color: "purple",
+  description:"He is quick witted",
+  age: 50,
+  image: "assets/plum.png",
+  occupation:"Professor"
+}
+
+const missScarlet = {
+  firstName: "Cassandra",
+  lastName:"Scarlet",
+  color: "red",
+  description:"She is smart",
+  age: 25,
+  image: "assets/scarlet.png",
+  occupation:"Fashion designer"
+}
+
+const mrsPeacock = {
+  firstName: "Eleanor",
+  lastName:"Peacock",
+  color: "blue",
+  description:"She is verbal",
+  age: 58,
+  image: "assets/peacock.png",
+  occupation:"Author"
+}
+
+const mrMustard = {
+  firstName: "Jack",
+  lastName:"Mustard",
+  color: "yellow",
+  description:"He is dangerous",
+  age: 64,
+  image: "assets/mustard.png",
+  occupation:"Military"
+}
+
+// CREATE OBJECTS FOR ALL THE WEAPONS
 
 const rope = {
   name: "rope",
   weight: 10
 }
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const knife = {
+  name:"knife",
+  weight: 7
+}
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS.
+const candlestick = {
+  name: "candlestick",
+  weight: 5
+}
+
+const dumbbell = {
+  name: "dumbbell",
+  weight: 18
+}
+
+const poison = {
+  name: "poison",
+  weight: 6
+}
+
+const axe = {
+  name: "axe",
+  weight: 15
+}
+
+const bat = {
+  name: "bat",
+  weight: 12
+}
+
+const trophy = {
+  name: "trophy",
+  weight: 16
+}
+
+const pistol = {
+  name: "pistol",
+  weight: 12
+}
+
+
+// THE ROOMS
+
+const diningroom = "Dining Room";
+const conservatory = "Conservatory"; 
+const kitchen = "Kitchen";
+const study = "Study";
+const library = "Library";
+const billiardroom = "Billiard Room";
+const lounge = "Lounge";
+const ballroom = "Ballroom";
+const hall = "Hall";
+const spa = "Spa";
+const livingroom = "Living Room";
```

Comment:
If you are only  this as simple strings you can add it directly to your `rooms` variable. This means less code, which you always should aim for. Ex: 
```
const rooms = [
  "Dinning Room",
  "Conservatory",
];
```